index.js

```diff
@@ -54,12 +54,30 @@ exports.generateFdf = function(data) {
   return fdf;
 }
 
-exports.generatePdf = function(data, templatePath, callback) {
+exports.generatePdf = function(data, templatePath, extendArgs, callback) {
   var tempName       = temp.path({suffix: '.pdf'}),
       tempNameResult = temp.path({suffix: '.pdf'}),
       pdfPath        = isAbsolute(templatePath) ? templatePath : path.join(__dirname, templatePath);
 
-  child = spawn("pdftk", [pdfPath, "fill_form", "-", "output", tempName, "flatten"]);
+  // retrieve arguments as array
+  var args = [];
+  for (var i = 0; i < arguments.length; i++) {
+       args.push(arguments[i]);
+  }
```

Comment:
Anyway to just loop over arguments after data && templatePath? Would help in readability and you wouldn't have to have lines 68 & 69.